CHANGELOG.md

```diff
@@ -84,11 +97,14 @@ Other minor additions
   *-suc : m * sucℤ n ≡ m + m * n
   ```
 
-* Added new proof to `Data.Nat.Properties`:
+* Added new proofs to `Data.Nat.Properties`:
   ```agda
-  even≢odd : ∀ m n → 2 * m ≢ suc (2 * n)
-  0≢1+n    : ∀ {n} → 0 ≢ suc n
-  n<1+n    : ∀ {n} → n < suc n
+  even≢odd     : ∀ m n → 2 * m ≢ suc (2 * n)
+  0≢1+n        : ∀ {n} → 0 ≢ suc n
```

Comment:
We tend to leave implicit arguments out of types in the changelog.

Reply:
these implicits are removed